### ISSUE TITLE
Docs show incorrect import location

### ISSUE THREAD

**Issue by alexlafroscia** [kellysutton/ember-reset-scroll#1]
I noticed that the readme shows that you can import the mixin like this:

``` javascript
import ResetScroll from 'ember-reset-scroll/mixins/reset-scroll';
```

However, because the mixin is inside `app/` instead of `addon/`, the mixin is virtually merged into the tree for the user's app.  The above doesn't work, but something like this does: 

``` javascript
import ResetScroll from '../../mixins/reset-scroll';
```

Which was the intention, the documented behavior, or the actual behavior?

**Comment by kellysutton** [comment]
Not the intention. I'll have to look into it. I followed the ember-cli structure for setting up this addon, which means the first example should work.

I'll investigate!

**Comment by alexlafroscia** [comment]
Yeah, it's a pretty minor difference and easy to miss.  The first example would work if the mixin was in `addon/mixins/reset-scroll`.  By being inside `app/`, it doesn't get namespaced under the name of the addon, which seems to be what you were looking to do.  I'd be happy to send a PR for the fix, if you'd like.

**Comment by runspired** [comment]
@kellysutton you actually didn't follow the standard ember-cli addon structure is the issue.

**Comment by kellysutton** [comment]
Hi folks. Sorry about this. This addon was created with an old version of ember-cli. Pull Requests welcome!

**Comment by runspired** [comment]
@kellysutton will be sending you one, don't worry
